01/1E.hector_obs_T.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "Hector v. Observed Temperatures"
+author: "Peter Scully"
+date: "2024-06-04"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Getting Data
+
+To start, let's import the relevant libraries:
+
+```{r imports, message=FALSE, warning=FALSE}
+library(hector)
+library(ggplot2)
+library(dplyr)
+```
+
+We now need to import the historical data into a data frame
+```{r cars}
+obs_data <- 
+  read.csv(file.path(here::here(), "01", 
+                     "HadCRUT.5.0.2.0.analysis.summary_series.global.annual.csv"))
+colnames(obs_data) <- c("year", "temperature", "lower", "upper")
+
+obs_data
```

Comment:
This is more of an FYI and will depend on your preference 

If you want to look at the data frame is structure, `head(df)` will return the first few entries of the data frame. While it can be helpful to take a look at what the data looks like during development, it is not necessary to include this print statement in the markdown that represents the final analysis.